internal/openmetrics-exporter/alerts_collector.go

```diff
@@ -24,15 +24,33 @@ func (c *AlertsCollector) Collect(ch chan<- prometheus.Metric) {
 	}
 	al := make(map[string]float64)
 	for _, alert := range alerts.Items {
-		al[fmt.Sprintf("%s,%s,%s", alert.Severity, alert.ComponentType, alert.ComponentName)] += 1
+		al[fmt.Sprintf("%s,%d, %s,%s, %d, %s, %s, %s, %s",
+			alert.Action,
+			alert.Code,
+			alert.ComponentName,
+			alert.ComponentType,
+			alert.Created,
+			alert.Description,
+			alert.KBurl,
+			alert.Severity,
+			alert.Summary,
+		)] += 1
 	}
 	for a, n := range al { 
 		alert := strings.Split(a, ",")
```

Comment:
Descriptions can have commas which is annoying. It has to be a different character then “,” I used \n in the FA alerts but I don’t even think that’s a great split because of SMB could potentially include “\\nameoffilesystem” which has “\n”